assets/components/countdown/countdown.jsx

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+// ----- Imports ----- //
+import React, { Component } from 'react';
+
+
+// ---- Types ----- //
+type CountdownTime = {
+  t: number,
+  days: number,
+  hours: number,
+  minutes: number,
+  seconds: number,
+}
+
+type PropTypes = {
+  date: number[]
+};
+
+type StateTypes = {
+  countdown: CountdownTime
+}
+
+
+// ---- Helpers ----- //
```

Comment:
Just a small detail here, there is a [global folder for helpers](https://github.com/guardian/support-frontend/tree/master/assets/helpers), if you think that these helpers could be reuse in some other places maybe is better to place them inside the global folder. If these helpers are really specific to this component, it is ok to leave them here. It is up to you.